samples/rqd/blender/Dockerfile

```diff
@@ -30,5 +30,10 @@ RUN tar -xvf blender.tar.xz \
 
 RUN rm blender.tar.xz
 
+# Add Blender path as environment variable
+ENV PATH=$PATH:${BLENDER_INSTALL_DIR}
+
+RUN echo "RQD_USE_PATH_ENV_VAR=true" >> /etc/opencue/rqd.conf
```

Comment:
This assumes the file already exists and has the `[Override]` heading which is required: https://www.opencue.io/docs/other-guides/configuring-opencue/#rqd

Looking at [the main RQD Dockerfile](https://github.com/AcademySoftwareFoundation/OpenCue/blob/226a27dbd9c3bdbc9effdddd03a4904bfa02824d/rqd/Dockerfile) I don't see anything that creates this file. I think we need to create it here?

This could be changed to:

```
RUN echo "[Override]" > /etc/opencue/rqd.conf
RUN echo "RQD_USE_PATH_ENV_VAR=true" >> /etc/opencue/rqd.conf
```

Reply:
Actually, I see that #1311 does create the config file in the main Dockerfile. You can ignore this comment.

Reply:
I added the .conf file to the base image Dockerfile in PR https://github.com/AcademySoftwareFoundation/OpenCue/pull/1311 and I assumed the same base image would be used here so only included the PATH-specific line since the conf file already exists.